src/prefect/deployments/deployments.py

```diff
@@ -793,6 +829,7 @@ async def apply(
                 work_pool_name=self.work_pool_name,
                 version=self.version,
                 schedule=self.schedule,
```

Comment:
I see we still have `schedule` (singular) in here, would it make sense to ensure we only have one or the other populated? Maybe even convert `schedule` to `schedules` if the plural version isn't supplied?

Reply:
Yeah, it would, I agree